server/src/main/java/io/spine/server/event/ERepository.java

```diff
@@ -150,20 +159,39 @@ private static CompositeColumnFilter timeFilter(EventStreamQuery query) {
             final ColumnFilter filter = lt(CREATED_TIME_COLUMN, timestamp);
             timeFilter.addFilter(filter);
         }
-        return timeFilter.build();
+
+        return buildFilter(timeFilter);
     }
 
-    private static CompositeColumnFilter typeFilter(EventStreamQuery query) {
+    private static Optional<CompositeColumnFilter> typeFilter(EventStreamQuery query) {
         final CompositeColumnFilter.Builder typeFilter = CompositeColumnFilter.newBuilder()
                                                                               .setOperator(EITHER);
         for (EventFilter eventFilter : query.getFilterList()) {
-            final String type = eventFilter.getEventType();
+            final String type = eventFilter.getEventType()
+                                           .trim();
```

Comment:
Can we have spaces in the `eventType` field?

Reply:
Done.